main.js

```diff
@@ -0,0 +1,23 @@
+var nameInput = document.querySelector('.expense-name');
+var amountInput = document.querySelector('.expense-amount');
+var button = document.getElementById('add-expense');
+var tableList = document.querySelector('tbody');
+
+
+
+var addAmtValue = function() {
+  var newTr = document.createElement('tr');
+  var newTd = document.createElement('td');
+  var newTd2 = document.createElement('td');
+  tableList.appendChild(newTr);
+  newTr.appendChild(newTd2);
+  newTr.appendChild(newTd);
+  newTd.textContent = amountInput.value;
```

Comment:
If you need to create a running tally of expense totals when do you think you'd have access to that data? In this function you're grabbing `amountInput.value` so that's probably a good start.